tests/src/performance/linkbench/scripts/clone.cmd

```diff
@@ -26,30 +26,30 @@ mkdir 2.0.0
 powershell -noprofile -executionPolicy RemoteSigned wget  https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.ps1 -OutFile dotnet-install.ps1
 powershell -noprofile -executionPolicy RemoteSigned -file dotnet-install.ps1 -InstallDir 1.0.0
 powershell -noprofile -executionPolicy RemoteSigned -file dotnet-install.ps1 -Channel master -InstallDir 2.0.0
-if not exist %__dotnet1% set EXITCODE=1
-if not exist %__dotnet2% set EXITCODE=1
+if not exist %__dotnet1% set EXITCODE=1 && echo DotNet.1.0.0 uninstalled
+if not exist %__dotnet2% set EXITCODE=1 && echo DotNet.2.0.0 uninstalled
 cd ..
 exit /b 
 
 :HelloWorld
 mkdir HelloWorld
 cd HelloWorld
-dotnet new console
+%__dotnet2% new console
```

Comment:
For safety, could you prefix this command with `call` (in the case the variable becomes a batch script and does not exit properly).